content/applications/finance/accounting/taxes/taxcloud.rst

```diff
@@ -2,150 +2,10 @@
-ID** you took note of earlier under the :guilabel:`Default Value (JSON format)` field.
-
-Select the company for which this configuration should apply under the :guilabel:`Company` field and
-click :guilabel:`Save`.
-
-.. image:: taxcloud/user-defaults-complete-configuration.png
-   :align: center
-   :alt: Example of a User-defined Defaults configuration
-
-Automatically detect the fiscal position
-========================================
-
-Sales taxes are calculated in Odoo based on :doc:`fiscal positions <fiscal_positions>`. A fiscal
-position for the United States is created when enabling TaxCloud.
-
-You can configure Odoo to automatically detect to which customers the fiscal position should be
-applied. To do so, go to :menuselection:`Accounting Dashboard --> Configuration --> Accounting:
-Fiscal Positions` and select :guilabel:`Automatic Tax Mapping (TaxCloud)`. Enable :guilabel:`Detect
-Automatically` and then :guilabel:`Save`.
-
-.. image:: taxcloud/fiscal-position-detect.png
-   :align: center
-   :alt: Detect Automatically setting on the TaxCloud fiscal position
-
-Now, this fiscal position is automatically set on any order or invoice if the customer country is
-*United States*. This triggers the automated tax computation.
-
-.. note::
-   To get the sales taxes on a sales order, confirm it or click the :guilabel:`Update Taxes` button
-   next to :guilabel:`Add Shipping`.
-
-Interaction with coupons and promotions
-=======================================
-
-If you use the **Coupon** or **Promotion Programs**, the integration with TaxCloud might behave
-unexpectedly. Indeed, as TaxCloud does not accept lines with negative amounts as part of the tax
-computation, the amount of the lines added by the promotion program must be deduced from the total
-of the lines it impacts.
-
-.. important::
-   This means, amongst other complications, that orders using coupons or promotions with a TaxCloud
-   fiscal position **must** be invoiced completely - you cannot create invoices for partial
-   deliveries, etc.
-
-Another unexpected behavior is possible. For example, you sell a product for which you have a
-promotion program that provides a 50% discount. If the product's tax rate is 7%, the tax rate
-computed from the TaxCloud integration displays 3.5%. This happens because the discount is included
-in the price sent to TaxCloud. However, in Odoo, the discount is on another line entirely. Still,
-the tax computation is correct. Indeed, a 3.5% tax on the full price is the equivalent of a 7% tax
-on half the price, but this might be unexpected from a user point of view.
+   The Odoo TaxCloud integration has begun its decommissioning, starting in Odoo 17. New
+   installations are prohibited in Odoo 17, and in Odoo 18, the TaxCloud module(s) won't exist at
+   all. Odoo recommends the use of the Avatax platform.
 
-.. seealso::
-   :doc:`fiscal_positions`
+   .. seealso::
+      :doc:`avatax`
```

Comment:
Would separate the **seealso** from the **warning** so it's not embedded visually.
```suggestion
.. seealso::
   :doc:`avatax`
```